src/pages/trade/model/trace.ts

```diff
@@ -1,5 +1,5 @@
 import { Trace } from '@/shared/api/server/book/types.ts';
-import { round } from '@/shared/utils/numbers/round.ts';
+import { round } from '@penumbra-zone/types/round';
```

Comment:
Moved `round` util to `@penumbra-zone/types` as it was needed in the UI component. Also augmented there with more features and tests.